compose/ui/ui/src/uikitMain/parse_crowdin.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+#
```

Comment:
Let's add a description why this script is needed, and step by step usage of it

Reply:
Done